### PR TITLE
[#952] Show Members of a Course on a Nation (connects #952)

### PR DESCRIPTION
**Pull request by razu9861**

### CODE REVIEW

**Review by nirojdyola — APPROVED** [review]

**Review by lmmrssa — COMMENTED** [review]
not sure will it work in case of community. please test it once.

**Comment by dogi** [comment]
pictures and wrong issue ;)

**Comment by razu9861** [comment]
screen shot 2017-09-13 at 10 25 44 am
screen shot 2017-09-13 at 10 26 05 am
screen shot 2017-09-13 at 10 26 14 am

**Comment by razu9861** [comment]
@dogi test failing at delete report because this branch was created before the tests were fixed.

**Comment by lmmrssa** [comment]
@razu9861 rebase this with dev and force push once then to pass the test.

**Comment by lmmrssa** [comment]
still there are changes which should not be in pull request such as revision number and changelog

**Comment by razu9861** [comment]
@dogi Test failing at language selection, where this PR has nothing to do with language selection

File: databases/membercourseprogress.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ ddoc.views = {
       }
     }
   },
-  GetMemberByCourseID: {
+   GetMemberByCourseID: {
```
Comment (lmmrssa) [inline databases/membercourseprogress.js]: this is not the change just indentation issue

**Pull request merged**